src/bentoml/_internal/frameworks/sklearn.py

```diff
@@ -163,7 +165,8 @@ def save_model(
         context=context,
         signatures=signatures,
     ) as bento_model:
-        joblib.dump(model, bento_model.path_of(MODEL_FILENAME))
+        with open(bento_model.path_of(MODEL_FILENAME), "wb") as f:
```

Comment:
What is the motivation for this?

Reply:
Was solving a community member's problem. In sklearn if you have a custom defined(not the built in ones) pipeline step, it will require to have the source code/module to be present when you load the model if we use `joblib` to serialize the pipeline. 

using cloudpickle solves the problem, like how pickable_model uses cloudpickle and many of our other integrations